packages/instantsearch-ui-components/src/components/chat/ChatMessage.tsx

```diff
@@ -297,27 +246,42 @@ export function createChatMessageComponent({
         <div className={cx(cssClasses.container)}>
           {hasLeading && (
             <div className={cx(cssClasses.leading)}>
-              {LeadingComponent ? (
-                <LeadingComponent />
-              ) : (
-                AvatarComponent && <AvatarComponent />
```

Comment:
did the avatar get removed on purpose? we don't have it in the design yet so good either way.

Reply:
I think it's a leftover from the Satellite implementation where we have an `Avatar` component. However, since we don't have that component in IS, providing the `leadingComponent` prop should be sufficient to add an avatar or whatever you want before the message.